### DISCUSSION TITLE
When will the 5.0 version be released?

### DISCUSSION THREAD

**Discussion by nylqd** [vesoft-inc/nebula#5866] [Q&A]
We are very much looking forward to the official support for isoGQL, and we anticipate its release as soon as possible.

**Comment by dutor** [comment]
Thanks for your attentions.
The GQL support is in the testing phase now, and the releasing will be in June or so. Please stay tuned.

**Reply by nylqd** [reply]
[Announcing the RC Release of NebulaGraph Enterprise v5.0: Delivering Full-Fledged GQL Support](https://www.nebula-graph.io/posts/rc_release_of_nebulagraph_enterprise_v5.0)

I learned that you have already released 5.0 RC for Enterprise, is there a timeline for the community version?

**Reply by hbriese** [reply]
Any update on GQL support for the community edition?

**Reply by wey-gu** [reply]
> Any update on GQL support for the community edition?

The cloud version that's accessible was just released!

We are working hard on a community version, but cannot guarantee a timeline.

**Reply by nylqd** [reply]
> > Any update on GQL support for the community edition?
> 
> The cloud version that's accessible was just released!
> 
> We are working hard on a community version, but cannot guarantee a timeline.

The community version hasn't seen any major updates in a year. Wondering if the hold-up on v5 is tech-related or more of a business call.
